src/main/java/uk/gov/hmcts/cmc/claimstore/services/ccd/callbacks/caseworker/PaperResponseAdmissionCallbackHandler.java

```diff
@@ -65,7 +65,7 @@ public class PaperResponseAdmissionCallbackHandler extends CallbackHandler {
     private final DocAssemblyTemplateBodyMapper docAssemblyTemplateBodyMapper;
     private final String paperResponseAdmissionTemplateId;
     private final UserService userService;
-    private final DocumentManagementService documentManagementService;
+    private final UnsecuredDocumentManagementService unsecuredDocumentManagementService;
```

Comment:
Inject with interface reference DocumentManagementService here instead of implementation class, as only one of the implementation is going to be available based on the config flag.